packages/kbn-config/src/env.ts

```diff
@@ -119,6 +119,7 @@ export class Env {
       rootDir: this.homeDir,
       oss: options.cliArgs.oss,
       examples: options.cliArgs.runExamples,
+      thirdParty: true,
```

Comment:
Why was this option necessary?

Reply:
I've added that extra option in order to be able to exclude the paths `rootDir/plugins, rootDir/../kibana-extra` as we don't want to run circular dependencies check on those

Reply:
Chatted with Tiago - this option is actually not required, so he is going to remove it until that changes.

Reply:
Alright, I've pushed the change into the PR! 😃